### ISSUE TITLE
Safari Release Possible?

### ISSUE THREAD

**Issue by belodelokelo** [sblask/webextension-skip-redirect#300]
### New Issue Checklist (tick off like this: `[x]`)

- [ x] I checked existing issues (open and closed) for possible duplicates
- [x] I can reproduce the problem on the latest stable version (not nightly!)
- [ ] If you have a problem with a specific website: adding the domain to your no-skip-urls-list does not solve the problem
- [ ] The issue is not about adding entries to the default no-skip-urls-list

### What is your browser?

- [ ] Firefox
- [ ] Chrome

### What is your operating system?

- [ ] Linux
- [ x] Mac
- [ ] Windows

**Comment by sblask** [comment]
I am not using Safari so I have no idea. Can't you get extensions from the Chrome Webstore?

**Comment by superevr** [comment]
> I am not using Safari so I have no idea. Can't you get extensions from the Chrome Webstore?

Safari and iOS require extensions to be tied to an App, so there's some building that has to be done first.

For the most part, the github source of this extension can be converted to an App Extension with XCode:
```xcrun safari-web-extension-converter /path/to/extension```
https://developer.apple.com/documentation/safariservices/safari_web_extensions/converting_a_web_extension_for_safari

But there's a number of incompatibilities that have to be worked through to get the extension to actually work. Unfortunately, it's a little above my head.

I would appreciate the effort though. Skip Redirect is the perfect extension for users of ad blockers like piHole. In that scenario, clicking a link with trackers in it leads to a blackhole. But Skip Redirect fixes that, leading directly to the content.

**Comment by sblask** [comment]
It's not going to happen from my side.

So you are using an extension and want to use Skip Redirect to undo what it does? Why not just not use the extension in the first place? Also there is no guarantee that would work because there is no defined order in which webextensions are called
